api/admin/get_locales.js

```diff
@@ -1,12 +1,23 @@
 /* eslint-disable no-underscore-dangle */
-
-const { getLocales } = require('../../lib/storage');
+const Boom = require('@hapi/boom');
+const storage = require('../../lib/storage');
 
 module.exports = () => ({
   method: 'GET',
-  config: {
-    auth: 'jwt'
+  options: {
+    auth: {
+      strategies: ['jwt'],
+      scope: ['profile']
```

Comment:
Any context on these scope checks being added?

Reply:
It's all profile related information. Dashboard admin's will always have `openid, email, profile` as it's hard coded for them on their JWTs. Figured adding the profile scope onto this endpoint adds an extra layer of security even if those scopes are still hard coded for dashboard admins